src/Playground.res

```diff
@@ -1342,24 +1342,29 @@ module App = {
     let (count, setCount) = React.useState(() => 0)
     let (username, setUsername) = React.useState(() => "Anonymous")
 
-    <div>
+    <div className="p-4">
       {React.string("Username: ")}
       <input
+        className="bg-gray-50 border border-gray-300 "
         type_="text"
         value={username}
-        onChange={evt => {
-          evt->ReactEvent.Form.preventDefault
-          let username = (evt->ReactEvent.Form.target)["value"]
+        onChange={event => {
+          event->ReactEvent.Form.preventDefault
+          let eventTarget = event->ReactEvent.Form.target
+          let username = eventTarget["value"]
           setUsername(_prev => username)
         }}
       />
       <button
+        className="px-4 bg-blue-600 text-white"
```

Comment:
It makes the example slightly more complex, but I believe it yields a better result.